client/components/suggestions/index.jsx

```diff
@@ -179,17 +180,15 @@ class Suggestions extends Component {
 											key={ originalIndex }
 											hasHighlight={ index === this.state.suggestionPosition }
 											query={ query }
+											onMount={ () =>
+												this.props.onSuggestionItemRender( {
+													suggestionIndex: originalIndex,
+													index,
+												} )
+											}
 											onMouseDown={ () => this.handleMouseDown( originalIndex ) }
 											onMouseOver={ () => this.handleMouseOver( index ) }
 											label={ label }
-											railcar={
-												railcar && {
-													...railcar,
-													railcar: `${ railcar.id }-${ originalIndex }`,
-													fetch_position: originalIndex,
-													ui_position: index,
-												}
-											}
```

Comment:
This received a `railcar` prop, transformed it and passed it down to where analytics was actually sent.

Rather than handle this here, it's pushed up to the `SearchSuggestion` component.